### ISSUE TITLE
Still working now?

### ISSUE THREAD

**Issue by franklinyu** [gitsense/insight#20]
I have just installed the extension, but didn’t see the data as in screenshot. Is it still working? Current GitHub layout may be different from the one last year.

**Comment by gitsense** [comment]
Yeah the extension is currently in limbo and has not been maintained properly.  This extension will probably be obsoleted in the near future, but I'll keep the code up, since others might learn something from it.
